docs/source/running-autometa.rst

```diff
@@ -30,21 +24,21 @@ function.
 .. code-block:: python
     :linenos:
 
-    python3
-    import autometa.common.external.samtools
-    autometa.common.common.external.samtools.sort(sam = <path/to/sam/file>, out=<path/to/output/file>, npoc = 4)
+    from autometa.common.external import samtools
+
+    samtools.sort(sam=<path/to/sam/file>, out=<path/to/output/file>, nproc=4)
 
 
 Data preparation
 ================
 
-Before you run Autometa, you need to have assembled your shotgun metagenome. 
-We recommend using MetaSPAdes (part of the SPAdes_ package) after removing Illumina 
+Before you run Autometa, you need to have assembled your shotgun metagenome.
+We recommend using MetaSPAdes (part of the SPAdes_ package) after removing Illumina
 adaptors with Trimmomatic_.
 
-Note that if you use SPAdes or something else that names contigs like 
-this: ``NODE_1_length_319818_cov_8.03695`` then Autometa can use the coverage 
-information in the contig names. If you have used another assembler, then 
+Note that if you use SPAdes or something else that names contigs like
+this: `` NODE_235_length_161117_cov_116.709`` then Autometa can use the coverage
```

Comment:
NODE_235_length_161117_cov_116.709 is not outputting the way we want it

Reply:
Ah, looks like there is an extra space at the beginning. I'll trim this.